packages/app/hooks/useCourse.tsx

```diff
@@ -15,7 +15,7 @@ export function useCourse(cid: number): UseCourseReturn {
     data: course,
     error: courseError,
     mutate: mutateCourse,
-  } = useSWR(cid && `/api/v1/courses/${cid}`, async () => API.course.get(cid));
```

Comment:
I am not sure if we should remove this, why couldn't we use a limited course info endpoint?